liquid/intranet/caffeine_manager/soda/views.py

```diff
@@ -10,10 +10,10 @@
 
 def allSodas(request):
     sodas=Soda.objects.all().order_by('-dispensed', 'name')
-    # vend = request.user.get_vending()
+    vend = request.user.get_vending()
     for s in sodas:
-        s.votedFor= 0 #(vend.votes.filter(id=s.id).count() == 1)
```

Comment:
These changes were made earlier as part of prod-debugging. (They disable existing functionality, but don't 'break' anything per se.)